conf/tax_years.conf

```diff
@@ -16,13 +16,31 @@ latest-tax-year-go-live-date
     {
     day = 06
     month = 04
-    year = 2023
+    year = 2024
```

Comment:
I'm not sure if this should be changed right now, I think in the past we changed this separately and then released the change on April 6th

Reply:
hey! we should be safe to do so in a single change. Following in TaxYearService reflects what it's used for and is needed to make sure we return correct list of tax year configs. Thanks
https://github.com/hmrc/cgt-property-disposals/blob/main/app/uk/gov/hmrc/cgtpropertydisposals/service/returns/TaxYearService.scala#L54